src/main/groovy/com/calgaryscientific/gradle/VeracodeTask.groovy

```diff
@@ -37,6 +37,12 @@ import com.veracode.apiwrapper.wrappers.ResultsAPIWrapper
 @CompileStatic
 abstract class VeracodeTask extends DefaultTask {
     abstract static final String NAME
+    VeracodeAPI veracodeAPI
+    VeracodeSetup veracodeSetup
```

Comment:
what's veracodeSetup?

Reply:
The extension point we use to setup the plugin: https://github.com/calgaryscientific/veracode-gradle-plugin/blob/master/src/main/groovy/com/calgaryscientific/gradle/VeracodeSetup.groovy

https://github.com/calgaryscientific/veracode-gradle-plugin#apply-the-plugin-into-your-project

In that extension, we define the `filesToUpload` field that the upload task requires.